research/templates/bash_template.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/bash
```

Comment:
Maybe a quick comment on this.

I usually use `#!/usr/bin/env bash` as it is more versatile and compatible with a wider range of systems, as well as take into account the current environment, which can save a few headaches. 

Now there are also disadvantages to this, which might not make it desirable in our context, but I though I might as well throw it out there and let you guys decide.

Ref: https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash